.github/workflows/build.yml

```diff
@@ -371,7 +376,47 @@ jobs:
           artifacts: |
             envelopes/*
           body: |
-            Envelopes for Toit SDK ${{ github.event.inputs.toit-version }}
+            Envelopes and partition tables for Toit SDK ${{ github.event.inputs.toit-version }}
+          allowUpdates: true
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.event.inputs.toit-version }}
+          commit: main
+
+  release-partition-tables:
+    name: "Release partition tables - ${{ github.event.inputs.toit-version }}"
+    if: ${{ github.event.inputs.build-tables == 'true' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Show inputs
+        run: echo "${{ toJSON(github.event.inputs) }}"
+
+      - uses: actions/checkout@v4
+
+      - name: Set the git version
+        run: |
+          echo "TOIT_GIT_VERSION=${{ github.event.inputs.toit-version }}" >> $GITHUB_ENV
+
+      - name: Prepare partition tables
+        shell: bash
+        run: |
+          mkdir -p build
+          shopt -s nullglob
+          for csv_file in partition-tables/*/partitions.csv; do
```

Comment:
Let's put these in partitions/*/partitions.csv - maybe even esp32/partitions/*/partitions.cvs.

Reply:
done.